apps/web/src/data/ecosystem.json

```diff
@@ -7,6 +7,30 @@
     "category": "consumer",
     "subcategory": "nft"
   },
+  {
+    "name": "DIMO",
+    "description": "DIMO is connected car platform for developers, enabling user-authorized access to vehicle data to create new types of games, social apps, leaderboards, dashboards, and more. For vehicle owners, the DIMO Mobile app unlocks your car’s data providing valuable vehicle health insights, personalized service offers ,and weekly rewards. DIMO Mobile is your all-in-one app for maximizing your car’s potential and helping you drive smarter.",
+    "url": "https://dimo.co",
+    "imageUrl": "/images/partners/dimo.png",
+    "category": "infra",
+    "subcategory": "developer tool"
+  },
+  {
+    "name": "SynFutures",
+    "description": "SynFutures (F) is a leading decentralized exchange (DEX) and full-stack financial infrastructure powering the future of trading.",
+    "url": "https://www.synfutures.com/",
+    "imageUrl": "/images/partners/synfutures.png",
+    "category": "defi",
+    "subcategory": "dex"
+  },
+  {
+    "name": "Vertex",
+    "description": "Vertex is an orderbook DEX offering spot, perpetuals, and money markets with unified cross-margin. Enjoy lightning-fast speeds, deep liquidity, low fees, cross-chain deposits, and multiple collaterals.",
+    "url": "https://link.vertexprotocol.com/base",
```

Comment:
just noting US is restricted, shouldn't matter 

<img width="419" alt="vertex" src="https://github.com/user-attachments/assets/d744f12e-6853-4866-8050-b9d487255637" />